src/odemis/acq/stitching/_registrar.py

```diff
@@ -68,17 +68,17 @@ def getPositions(self):
         """
         returns:
         tile_positions (list of N tuples of 2 floats)): the adjusted position in X/Y for each tile, in the order they were added
-        dep_tile_positions (list of N tuples of tuples of 2 floats): the adjusted position for each dependent tile 
+        dep_tile_positions (list of N tuples of tuples of 2 floats): the adjusted position for each dependent tile
         (in the order they were passed)
         """
         return self.tile_pos, self.dep_tiles_pos
 
 
 class ShiftRegistrar(object):
     """
-    Locates the position of the image relative to the previous top tiles (horizontally and vertically) 
-    by using cross-correlation. The cross-correlation is done using just the part of the images which are 
-    supposed to be overlapping. In case the cross-correlation doesn't work (based on a couple of simple tests), 
+    Locates the position of the image relative to the previous top tiles (horizontally and vertically)
```

Comment:
Is an image not made up of the tiles? I am not familiar with the code, so do you want to locate the position of a tile (and not an image) relative to the previous tiles?

Reply:
Here "image" and "tile" are used interchangeably. Admittedly, it'd be less confusing to use always "tile". 
@Kleijwegt You see why you shouldn't fix the formatting at the same time as the code: people start talking about things completely unrelated ;-)
From my point of view, I'm fine whether you want to also start fixing/improving docstrings... but then do it in a separate commit!